crates/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -812,11 +822,130 @@ fn new_session<'state, 'vm, S: StorageView>(
     )
 }
 
-/// Load an input object from the state_view
-fn load_object<S: StorageView>(
-    vm: &MoveVM,
-    state_view: &S,
+/// Set the link context for the session from the linkage information in the `package`.
```

Comment:
I'm actually considering going the other way and ditching the invariant violations when a move call is made without linkage information set.  This is so that I can support backwards compatibility for when the upgrades feature flag is turned off